scripts/test.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 set -e
 
-# run relayer docker build script
-source $(dirname $0)/testDockerRelayer.sh
+npx nx run-many --target=test --all --parallel=false
 
-npx nx run-many --target=test --all --parallel=false
\ No newline at end of file
+# run relayer docker build script
+. $(dirname $0)/testDockerRelayer.sh
```

Comment:
This seems to be wrong. `.` means `source` (including environment variables from the given script), not running a script.

you probably want to do

```suggestion
./$(dirname $0)/testDockerRelayer.sh
```

Or just

```suggestion
$(dirname $0)/testDockerRelayer.sh
```